test/e2e/specs/editor/various/draggable-blocks.spec.js

```diff
@@ -111,38 +111,34 @@ test.describe( 'Draggable block', () => {
 		await dragHandle.hover();
 		// Start dragging.
 		await page.mouse.down();
-		await dragHandle.hover( {
-			// Move the dragged handle a little bit (to the top left of the handle's
-			// original location) to trigger the dragging chip to appear.
-			position: { x: 0, y: 0 },
-			// Bypass the "actionability" checks because the handle will disappear.
-			force: true,
-		} );
-
-		await expect(
-			page.locator( 'data-testid=block-draggable-chip >> visible=true' )
-		).toBeVisible();
 
 		// Move to and hover on the bottom half of the paragraph block to trigger the indicator.
 		const secondParagraph = page.locator(
 			'role=document[name="Paragraph block"i] >> text=2'
 		);
 		const secondParagraphBound = await secondParagraph.boundingBox();
 		await page.mouse.move(
-			secondParagraphBound.x + secondParagraphBound.height * 0.5,
+			secondParagraphBound.x,
 			secondParagraphBound.y + secondParagraphBound.height * 0.75
 		);
-		await secondParagraph.hover( {
```

Comment:
@kevin940726 It seems inconsistent that the first test uses only `hover` and the second uses only `mouse.move`. What's the reason behind that?

It'd be good to find a fairly standardized approach so that adding more tests later on is easier.

Reply:
Partly because the second test requires us to calculate the position based on the block's height. Another reason is that `hover` by default will check for the element's "actionability", but for whatever reason the second test will and only will fail in headless mode, I'm not sure why yet. We can use `force` to bypass the check though, but it might be more difficult to read for people not that familiar with Playwright's advanced API.

I agree there's inconsistency though, maybe we should both use `mouse.move`?

Reply:
In my testing the second test failed unless using both hover and move. It might be that the first test was only passing because the mouse is already in the right position to drop above the first block when dragging starts.